src/app/tests/suites/certification/Test_TC_WNCV_3_2.yaml

```diff
@@ -223,7 +223,7 @@ tests:
       arguments:
           values:
               - name: "ms"
-                value: 3000
+                value: 5000
```

Comment:
This changes certification tests ... is this safe?

@cjandhyala @raju-apple ?

Reply:
I don't see the test spec has defined the delay here, currently, it takes the window 5 seconds to fully move from one side to another. Delay 3s will also works for followup checks, but the windows is in operating in those checks

Reply:
3a2 is a extra step only in the yaml, not in the current test spec.
https://github.com/CHIP-Specifications/chip-test-plans/blob/master/src/cluster/window_covering.adoc

Reply:
at the end i think those minimum times should maybe be part of PIXITs in test plan

Reply:
these are not waiting for a full motion of an actuator but for a slight move only 
how long a real device react i would say depending of the radio thread + firmware delay + mechanical delays
this why i think next step would be to add a minimum waiting time as some PIXIT

Reply:
Yes , we anyway do not support PIXIT for SVE-2 , so lets get this change also in post SVE-2

> > > > Lets not make this change now as it will affect the cert tests and SVE-2 , for now if Darwin CI is flaky can disable this test from the Darwin CI.
> > > 
> > > 
> > > I don't see any of changes in this PR requires changes in test spec
> > 
> > 
> > SVE-2 is almost done and folks have submitted/reviewed results for most of the TCs, (Didn’t see any issues raised for this TC) any script changes now would trigger a lot of churn, so if this test continues to be flaky in Darwin CI, we should disable it in Darwin CI for now and after SVE-2 is done can fix it as needed .
> 
> But the changes within this PR does not require any changes in WNCV TCs, all adjustments are within the specified range of TCs

The right fix would be to add PIXIT implementation in the test plan for such scenarios and since we don't have it yet for SVE-2 , for now we should just disable in Darwin CI if it continues to be flaky and not try to change the "working" timeout/sleeps just to make it less flaky on Darwin. cc @bzbarsky-apple @andy31415